test/conftest.py

```diff
@@ -168,7 +168,7 @@ def pudl_out_ferc1(live_dbs: bool, pudl_engine: sa.Engine, request) -> PudlTabl:
     """Define parameterized PudlTabl output object fixture for FERC 1 tests."""
     if not live_dbs:
         pytest.skip("Validation tests only work with a live PUDL DB.")
-    return PudlTabl(pudl_engine=pudl_engine, freq=request.param)
+    return PudlTabl(pudl_engine=pudl_engine, freq=request.param, table_source="parquet")
```

Comment:
Redirect all the PudlTabl objects used in our tests (the only place it's still actually used) to read data from Parquet outputs instead of SQLite.  There are a number of these objects defined in other fixtures for various different testing purposes.  I tried to change them all.

Reply:
If we switch to `PudlTabl` reading from Parquet all the time, I'll remove these.